### ISSUE TITLE
How can we  apply  a few-shot  using  image with mask  and text prompt 

### ISSUE THREAD

**Issue by forever-rz** [timojl/clipseg#22]
Thanks for your contribution,I'm still a bit confused on how to do a few- shot?
Is it just a matter of continuing to train  using the training code or  or fine-tune?

**Comment by timojl** [comment]
For a few-shot setting, I recommend running the pre-trained model multiple times (i.e. once for every few-shot sample) and averaging the predictions. This way, you do not need to re-train.
